src/test/java/org/hydev/mcpm/client/installer/InstallSetUp.java

```diff
@@ -0,0 +1,56 @@
+package org.hydev.mcpm.client.installer;
+
+import org.hydev.mcpm.client.Downloader;
+import org.hydev.mcpm.client.database.fetcher.DatabaseFetcher;
+import org.hydev.mcpm.client.database.tracker.SuperPluginTracker;
+import org.hydev.mcpm.client.display.presenters.InstallPresenter;
+import org.hydev.mcpm.client.injector.PluginLoader;
+import org.hydev.mcpm.client.local.LocalDatabaseFetcher;
+import org.hydev.mcpm.client.local.SuperLocalPluginTracker;
+import org.hydev.mcpm.client.search.SearchInteractor;
+import org.hydev.mcpm.client.search.SearchPackagesBoundary;
+import org.hydev.mcpm.utils.ColorLogger;
+
+import java.net.URI;
+
+public class InstallSetUp {
+    private static final Downloader downloader = new Downloader();
+    private static final PluginLoader loader = null;
+    private static final URI host = URI.create("https://mcpm.hydev.org");
```

Comment:
what if we don't have internet

Reply:
I wonder how to address this issue, it seems like there's a need to get an empty database through install interactor.